jupyter-notebooks/ship-detector/01_ship_detector.ipynb

```diff
@@ -0,0 +1,304 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Detect ships in Planet data\n",
+    "This notebook demonstrates how to use Plant data to detect and count the number of ships using image data clipped from an AOI (in this case, a small area in the San Francisco Bay).\n",
+    "\n",
+    "Once ships are detected, the script will generate pngs of the ships and a png of input the image with outlines around the ships."
+   ]
```

Comment:
Suggestion: Change opening paragraph to "This notebook demonstrates how to detect and count objects in satellite imagery using algorithms from Python's [scikit-image](http://scikit-image.org/) library. In this example, we'll look for ships in a small area in the San Francisco Bay and generate a PNG of each ship with an outline around it.